src/mozilla_sec_eia/library/validation_helpers.py

```diff
@@ -65,17 +70,56 @@ def jaccard_similarity(
         value_col: Column to calculate Jaccard similarity on.
             Must be present in both dataframes.
     """
-    # fill nans to make similarity comparison more accurate
-    if (computed_df[value_col].dtype == float) and (
-        validation_df[value_col].dtype == float
-    ):
-        computed_df[value_col] = computed_df[value_col].fillna(999)
-        validation_df[value_col] = validation_df[value_col].fillna(999)
-    else:
-        computed_df[value_col] = computed_df[value_col].fillna("zzz")
-        validation_df[value_col] = validation_df[value_col].fillna("zzz")
     intersection = set(computed_df[value_col]).intersection(
         set(validation_df[value_col])
     )
     union = set(computed_df[value_col]).union(set(validation_df[value_col]))
     return float(len(intersection)) / float(len(union))
+
+
+def clean_ex21_validation_set(validation_df: pd.DataFrame):
+    """Clean Ex. 21 validation data to match extracted format."""
+    validation_df = validation_df.rename(
+        columns={
+            "Filename": "id",
+            "Subsidiary": "subsidiary",
+            "Location of Incorporation": "loc",
+            "Ownership Percentage": "own_per",
+        }
+    )
+    validation_df["own_per"] = validation_df["own_per"].astype(str)
+    validation_df["filename"] = validation_df["id"].apply(get_metadata_filename)
+    validation_df = clean_extracted_df(validation_df)
+    return validation_df
+
+
+def strip_down_company_names(ser: pd.Series) -> pd.Series:
```

Comment:
Before computing comparison metrics between the expected validation subsidiary column and the extracted subsidiary column, I strip off company name suffixes (i.e. "LLC") to make the comparison more accurate. The suffixes are kept on the actual subsidiary columns that are extracted into tables, this is just used for validation.